moveit_simple/src/robot.cpp

```diff
@@ -1526,6 +1535,36 @@ bool Robot::getIK(const Eigen::Affine3d pose, std::vector<double>& joint_point,
   return false;
 }
 
+void Robot::limitJointWindup(bool enabled)
+{
+  limit_joint_windup_ = enabled;
+}
```

Comment:
i don't like how ``getIK`` has a ``limit_joint_windup`` but this public function also exist. should be one or the other...

Reply:
See https://github.com/PickNikRobotics/moveit_simple/pull/3#discussion_r279870827.
It's a matter of what user level is accessing the function. A normal user only has access to the public function `getJointSolutions()` which uses `limit_joint_windup_` and calls `getIK()`. `getIK()` is protected and is free to be used with and without joint windup reduction, i.e. when sampling with symmetries (later PR).